src/phoenix/server/api/mutations/span_annotations_mutations.py

```diff
@@ -72,7 +72,7 @@ async def patch_span_annotations(
                         (
                             models.SpanAnnotation.annotator_kind,
                             annotation.annotator_kind.value
-                            if annotation.annotator_kind is not None
+                            if annotation.annotator_kind is not None and annotation.annotator_kind is not UNSET
```

Comment:
when annotator kind was not passed in (cannot be updated), we were getting an issue reading `value` off of UNSET type

Reply:
@anticorrelator can you fix up this part?